email-templates/E-conomic/DA/ONBOARDING.mjml

```diff
@@ -92,7 +92,7 @@
         		<mj-text font-size="14px"
             		     padding="0"
             		     font-family="Roboto, sans-serif">
-        						 Hej {{ signee_name }}.<br/><br/>Du har nu gennemført installationen af Straksindbetaling for {{ company_name }} i e-conomic og kan få betaling med det samme for fakturaer gennem Moneyflow.<br/><br/> Vedhæftet denne e-mail finder du hovedaftalen som du har underskrevet.<br/><br/>Når du laver og sender en faktura, har du nu mulighed for at aktivere Straksindbetaling for den specifikke faktura. 
+        						 Hej {{ signee_name }}.<br/><br/>Du har nu gennemført installationen af Straksindbetaling for {{ company_name }} i e-conomic og kan få betaling med det samme for fakturaer gennem Moneyflow.<br/><br/> Som noget helt særligt vi vil gerne tilbyde jer at bruge Straksindbetaling til halvdelen af prisen. Den første faktura er selvfølgelig stadig gratis.<br></br>I skal egentlig ikke gøre noget for at aktivere din rabat bortset fra at huske at slå Straksindbetaling til, når I opretter jeres næste faktura i e-conomic. <br></br>Vedhæftet denne e-mail finder du hovedaftalen som du har underskrevet.<br/><br/> 
```

Comment:
In `...stadig gratis.<br></br>...`, for example, did you intend a double space (`<br/><br/>`), or a single space with self-closing tag (`<br/>`)?

Reply:
I intended to do double space with <br></br>, fixed it now